pyzo/core/baseTextCtrl.py

```diff
@@ -25,41 +25,12 @@ def normalizePath(path):
     """Normalize the path given.
     All slashes will be made the same (and doubles removed)
```

Comment:
This function is used in a few other places, but I think its ok to reduce it like you did here.